File: fix_image.js
const sharp = require('sharp');
const path = require('path');
const fs = require('fs');

const imgPath = path.join(__dirname, 'src', 'assets', 'gallery', 'new-gallery-09.jpeg');
const outPath = path.join(__dirname, 'src', 'assets', 'gallery', 'new-gallery-09-temp.jpeg');

console.log('Starting...');
console.log('Input:', imgPath);
console.log('Exists:', fs.existsSync(imgPath));

// Use sharp to auto-rotate based on EXIF, then save
sharp(imgPath)
    .rotate() // auto-rotate based on EXIF orientation
    .jpeg({ quality: 95 })
    .toFile(outPath)
    .then(info => {
        console.log('Output info:', JSON.stringify(info));
        // Replace original with fixed version
        fs.copyFileSync(outPath, imgPath);
        fs.unlinkSync(outPath);
        console.log('Replaced original with auto-rotated version');
    })
    .catch(err => {
        console.error('Error:', err.message);
    });

File: fix_image.py
from PIL import Image
import os

path = os.path.join('src', 'assets', 'gallery', 'new-gallery-09.jpeg')
print(f"Opening: {path}")
img = Image.open(path)
print(f"Original size: {img.size}")

# Rotate 90 degrees clockwise (people's heads go from left to top)
rotated = img.rotate(-90, expand=True)
print(f"Rotated size: {rotated.size}")

rotated.save(path, quality=95)
print("Saved successfully!")
